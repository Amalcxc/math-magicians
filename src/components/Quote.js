import React from 'react';

function Quote() {
  return <div>
    <h4 className='quote'>
    Without mathematics, there’s nothing you can do.
    Everything around you is mathematics.
    Everything around you is numbers.
     — Shakuntala Devi, Indian writer and mental calculator
    </h4>
  </div>;
}

export default Quote;