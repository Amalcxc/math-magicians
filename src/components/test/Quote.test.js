import React from 'react';
import { render } from '@testing-library/react';
import Quote from '../Quote.js';

test('Renders Calculator Component', () => {
  render(<Quote />);
  const div = document.createElement('div');
  div.innerHTML = '<h4 className="quote"></h4>';
  expect(div).toMatchSnapshot();
});