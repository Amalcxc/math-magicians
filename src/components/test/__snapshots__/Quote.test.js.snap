// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Calculator Component 1`] = `
<div>
  <h4
    classname="quote"
  />
</div>
`;
